bindings/python/test/test_mongocrypt.py

```diff
@@ -261,6 +266,69 @@ def test_decrypt(self):
         self.assertEqual(decrypted, bson_data('command-reply.json'))
 
 
+class KeyVaultCallback(MockCallback):
+    def __init__(self, kms_reply=None):
+        super(KeyVaultCallback, self).__init__(kms_reply=kms_reply)
+        self.data_key = None
+
+    def fetch_keys(self, filter):
+        return self.data_key
+
+    def insert_data_key(self, data_key):
+        self.data_key = data_key
+        return BSON(data_key).decode()['_id']
```

Comment:
bson.decode()

Reply:
We can't use `bson.decode()` because it hasn't been released yet.

Reply:
Oh, right, since this lives outside of PyMongo this would make testing more difficult at the moment,